.gitignore

```diff
@@ -2,3 +2,4 @@
 .DS_Store
 bin/configlet
 bin/configlet.exe
+/exercises/testbox
```

Comment:
Your tests probably shouldn't be running within the exercises subdirectory.

Reply:
Can you explain what you mean?  To be clear, that folder is simply the TestBox framework which the users will need to install to run the tests.  That's what gets created by `box install` command in combination with the `box.json`.  I placed it there so all the exercises could share the TestBox installation but it's ignored because it shouldn't be part of the repo.  When a user wants to run the tests for a specific exercise, they'll `cd` into that specific exercise directory and run `box task run TestRunner` like I documented in my initial pull comment.

Reply:
What happens when exercism adds an exercise called `testbox`
(This may be a case of: "I'll worry about that if it ever happens.")

Reply:
Then that would cause problems :smile:  Based on our conversation, I'll probably just end up installing the TestBox library instead each and every exercise.  It seems like wasted files, but at the end of the day, it will be seamless to the user since the CLI task that runs the tests will just install it automatically.